client.lua

```diff
@@ -706,7 +705,7 @@ RegisterNetEvent('ox_inventory:createDrop', function(drop, data, owner, slot)
 		drops[drop] = data
 	end
 
-	if owner == PlayerData.id and invOpen and #(GetEntityCoords(PlayerData.ped) - data.coords) <= 1 then
+	if owner == GetPlayerServerId(PlayerData.id) and invOpen and #(GetEntityCoords(PlayerData.ped) - data.coords) <= 1 then
```

Comment:
Did I end up storing PlayerData.source? If not then add that during init and use here.